### DISCUSSION TITLE
@next/font + Tailwind docs or example

### DISCUSSION THREAD

**Discussion by corysimmons** [vercel/next.js#42484] [Help] [answered]
👋 Hello, I can't figure out how to get @next/font working with Tailwind. Are there any docs or examples anyone could point me towards, please?

**Comment by alvarlagerlof** [comment, marked as answer]
I wrote an article about it: https://alvar.dev/blog/tailwindcss-with-next-font

**Reply by corysimmons** [reply]
Awesome, thank you so much!

**Comment by leerob** [comment]
Docs have been updated: https://beta.nextjs.org/docs/optimizing/fonts#with-tailwind-css

**Reply by corysimmons** [reply]
Amazing, thank you Lee!
